server/services/parcelActionsService.js

```diff
@@ -1,7 +1,25 @@
-const actions = require('../../data/actions.json')
+const actionsService = require('./actionsService')
+const parcelService = require('./parcelService')
+const rulesEngine = require('../rules-engine')
 
 module.exports = {
   get: async function (parcelRef) {
-    return actions
+    const actions = await actionsService.get()
+    const returnActions = []
+    const parcelData = parcelService.getByRef(parcelRef)
+    for (const action of actions) {
+      rulesEngine.resetEngine()
+      let actionPassed = false
+      const successFunc = function () {
+        actionPassed = true
+      }
+      await rulesEngine.doEligibilityRun(action.rules, [parcelData], { }, successFunc)
+      if (actionPassed) {
+        const returnAction = Object.assign({}, action)
```

Comment:
this could be done in one line with:
` const { rules, ...returnAction } = action`